python/plugins/processing/algs/qgis/VariableDistanceBuffer.py

```diff
@@ -143,6 +143,18 @@ def name(self):
     def displayName(self):
         return self.tr("Variable distance buffer")
 
+    def shortDescription(self):
+        return self.tr(
+            "Assigns a color index to polygon features, so no adjacent polygons share the same color index."
```

Comment:
Good catch, Nyall. Would this be OK?
```suggestion
            "Computes a buffer area whose size is defined by an attribute."
```

Reply:
Sounds good. This algorithm is deprecated and hidden anyway, so I wouldn't worry too much about it.